### ISSUE TITLE
Keyboard a11y issues across nav layouts

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/foundation#81]
https://github.com/omeka-s-themes/foundation/issues/78 discusses issues with the menu pattern for accordion menus on this theme (applicable to both the off-canvas menu on smaller viewports and the vertical menu layout available via the full height column nav configuration). This issue focuses on larger keyboard accessibility issues across Foundation's menus on both the horizontal dropdown and full height column patterns.

For the horizontal dropdown:
- Screen reader function looks good here, although with keyboard alone, it is impossible to actually activate any menu item with a dropdown toggle (parent or child). However, they can be interacted with, opening dropdowns down through the nav structure (until reaching an item without a toggle that can be activated appropriately), so I think this is fundamentally an issue with the toggles and menu items being one and the same.
- For both screen reader and keyboard nav, dropdowns do not close automatically after tabbing away from them (unless a separate dropdown parent item is activated).

For the full height column:
- Some toggles are not visible, though they do function appropriately:
  <img width="468" height="333" alt="Full height column accordion menu. Some parent items with child items are missing toggles, while others are visible" src="https://github.com/user-attachments/assets/800a807c-6ed4-46f6-af74-3846009b4ca2" />
  This is true for all stylesheets.

True to both:
- On desktop, when the off-canvas menu is activated from a small enough viewport and the viewport is then widened without first closing the menu, the off-canvas menu remains open (so two types of nav menus appear on screen), and the screen gains horizontal scroll:
  <img width="1280" height="593" alt="Full screen with horizontal drop down menu and off-canvas menu, both visible on screen at the same time. There is a horizontal scrollbar at the bottom of the screen" src="https://github.com/user-attachments/assets/7f8c598b-dccf-444b-8c0b-8464be7768a0" />
  <img width="1280" height="595" alt="Full screen with full height column menu and off-canvas menu, both visible on screen at the same time. There is a horizontal scrollbar at the bottom of the screen" src="https://github.com/user-attachments/assets/5a4a8589-c812-4e37-8387-985ae71b7cb4" />
- Same issue as the theme on Classic, https://github.com/omeka/theme-foundation/issues/69: Focus can move outside of the off-canvas menu to elements on the page, not all of which are in view (and all of which are dimmed). I'm thinking it makes more sense to make all exterior content hidden while adding a close menu toggle directly into the off-canvas menu.
